doc/classes/CompressedTexture2D.xml

```diff
@@ -19,7 +19,7 @@
 	</methods>
 	<members>
 		<member name="load_path" type="String" setter="load" getter="get_load_path" default="&quot;&quot;">
-			The StreamTexture's file path to a [code].stex[/code] file.
+			The CompressedTexture's file path to a [code].stex[/code] file.
```

Comment:
I think this should be changed to `.ctex`.

Reply:
Fixed by https://github.com/godotengine/godot/pull/58965.